packages/providers/src.ts/fallback-provider.ts

```diff
@@ -351,9 +351,6 @@ async function getRunner(config: RunningConfig, currentBlockNumber: number, meth
     let provider = config.provider;
 
     switch (method) {
-        case "getBlockNumber":
-        case "getGasPrice":
```

Comment:
I think that we should leave `getGasPrice` since technically the method exists but it is not supported **yet**